samples/graph-app-installation-lifecycle/csharp/AppInstallation/AppInstallation.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Could you please add entry in main read me file under Graph?

Reply:
added link in Root ReadMe file also update app readMe